### DISCUSSION TITLE
Customization & Todo list

### DISCUSSION THREAD

**Discussion by Kr0sFyr3** [adrian-miasik/unity-pomodoro#116] [Ideas]
Open Source Project Plan
====
<p align="center">
<a href="https://github.com/adrian-miasik/unity-pomodoro" rel="Unity Pomodoro ">
<img src="https://raw.githubusercontent.com/adrian-miasik/unity-pomodoro/develop/UnityPomodoro/Assets/AdrianMiasik/Sprites/icon-unity-pomodoro.png" width=128 />
</a>
</p>

--- 

#### Required
> ###### 1. Adding a to do list that can be filled out and linked to a number of pomodoro's required to finish.
> ###### 2. The current app uses a predefined colour scheme. I would like to add the functionality for the user to change the color scheme according to his/her own wishes.

#### Optional
> ###### 3. Adding custom ringtones/alarms by the use of a youtube link or audio file on the system.
> ###### 4. Adding binaural beats for focus
> ###### 5. Adding an overlaying game like tamagotchi to help motivate to finish your todo list.

**Comment by adrian-miasik** [comment]
Heya @KingWazzack,

Good work so far on the features! 
I'm reaching out because I'd love to incorporate your color changes into the project upstream.

There's a couple things that need your attention to make this possible. I've left a comment on the relevant PR of a couple outstanding concerns. https://github.com/adrian-miasik/unity-pomodoro/pull/120#issuecomment-1622741120
